sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/IssueLocationCollector.cs

```diff
@@ -208,29 +291,50 @@ private IEnumerable<string> GetIssueIds(Match match)
 
         private int GetOffset(Match match)
         {
-            var sign = match.Groups["sign"];
             var offset = match.Groups["offset"];
-            if (!sign.Success && !offset.Success)
+            return offset.Success ? int.Parse(offset.Value) : 0;
```

Comment:
It would be nice to have `GetOffset` above `GetIssueIds` to have the similar methods together.